holidays/countries/india.py

```
@@ -164,6 +170,25 @@ class India(
         "OR",  # Orissa.
     )
 
+    janmashtami_optional_years = {2008, 2017}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)